src/js/page_sections/Menu/Menu.js

```diff
@@ -174,7 +171,13 @@ const PrivacyPolicyListItem = ({intl}) => (
 const MobileFooter = ({intl}) => (
   <div className="coa-Menu__mobile-footer">
     <p className="coa-Menu__footer-text d-lg-none">
-      {intl.formatMessage(i18nMessages.footer)}<ExternalLink to="https://austintexas.gov">austintexas.gov</ExternalLink>.
+      <FormattedMessage
+        id="Menu.MobileFooter.text"
+        defaultMessage="Alpha.austin.gov is a work in progress. For the full City of Austin website, visit {citySiteLink}."
```

Comment:
so a few things.
1. by changing the format here by making the link a parameter and part of the formatted message, you're ALSO changing the display of the links in all languages. you'll have to update the other language json files like you did by updating the en.json to include ` {citySiteLink}` as the end of the string.
2. this mobile menu footer text looks different than the arabic version which shows two links? The format should be the same.
3. Awhen you use FormattedMessage, it throws everything in a container span.

Reply:
Since the two `<FormattedMessage>`s are the same, any issue if they just use the same id?